src/main/flight/failsafe.c

```diff
@@ -47,7 +48,15 @@
 
 static failsafeState_t failsafeState;
 
-static failsafeConfig_t *failsafeConfig;
+failsafeConfig_t failsafeConfig;
+
+static const pgRegistry_t failsafeConfigRegistry PG_REGISTRY_SECTION =
```

Comment:
Maybe define macro that will hide these details. It will make future changes much easier:
```
PG_REGISTER(failsafeConfig, MSP_FAILSAFE_CONFIG, MSP_SET_FAILSAFE_CONFIG);
```